Library/ViewModels/ShareViewModel.swift

```diff
@@ -21,13 +21,6 @@ public struct ShareActivityCompletionData {
   }
 }
 
-/// These share types provide us access to knowing when the user successfully shares through that method,
-/// or when the user cancels.
-private let firstPartyShareTypes: [UIActivity.ActivityType] = [
```

Comment:
Why was this snippet deleted specifically?

Reply:
Wasn't being used, its private so it cannot be used anywhere but inside this file. Searched for references in this file and there were none.